events/mailer.go

```diff
@@ -171,18 +190,14 @@ func (e *Mailer) newBatchReapableEvent(rs []Reapable, tags []string) error {
 
 // Send an HTML email
 func (e *Mailer) send(to mail.Address, subject string, htmlBody *bytes.Buffer) error {
-	log.Debug("Sending email to: \"%s\", from: \"%s\", subject: \"%s\"",
-		to.String(),
-		e.Config.From.Address,
-		subject)
-
 	m := email.NewEmail()
 	m.From = e.Config.From.Address
 	m.To = []string{to.Address}
 	m.Bcc = e.Config.CopyEmailAddresses
 	m.Subject = subject
 	m.HTML = htmlBody.Bytes()
 
+	NewCountStatistic("reaper.emails.sent", []string{to.Address})
```

Comment:
I threw this in... good thing to track. Doesn't necessarily belong in this PR, whoops.